api/app/Console/Commands/CreateDigitalCareersPools.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\User;
+use App\Models\Pool;
+use App\Models\Skill;
+use App\Models\Classification;
+use Database\Helpers\ApiEnums;
+use Illuminate\Console\Command;
+use Carbon\Carbon;
+
+class CreateDigitalCareersPools extends Command
+{
+    // this command creates 32 pools, 4 classifications by 8 streams
+    // connected to a classification and 5 essential skills
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'create:digital_careers_pools';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create 32 pools';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        // create 32 pools
+        $ITLevels = [1, 2, 3, 4];
+        $poolStreams = ApiEnums::poolStreams();
+        $dateNow = Carbon::now();
+        $ownerId = User::all()->sole('sub', 'ilike', 'admin@test.com')['id']; // this for local testing, the one below for other environments
```

Comment:
it creates an array with the id instead of returning just the id
![image](https://user-images.githubusercontent.com/40485260/198698080-64f8701e-1cd2-4839-9ffc-4f81be87944b.png)

so I think you have to do something like this instead, so not much of a point to picking out just the id
`$ownerId = User::where('sub', 'ilike', 'admin@test.com')->sole(['id'])['id']`

Reply:
switched sole around a bit to this https://github.com/GCTC-NTGC/gc-digital-talent/pull/4467/commits/e7fb1fa68dd7ad770f1663f4a4cbee502866cc88

Reply:
My bad, I thought passing an array into sole would select only those columns for you. I must have misunderstood some documentation or example somewhere.